ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java

```diff
@@ -219,6 +219,8 @@ public CompletableFuture<?> start() {
 
     final CompletableFuture<?> resultFuture = new CompletableFuture<>();
     try {
+      NetworkUtility.checkIfPortIsAvailable(config.getPort());
```

Comment:
Think this check would be better to happen sooner in BesuCommand otherwise this will need to be added in to multiple services. Perhaps near where we do the checkPortClash() call in BesuCommand configure method?

Reply:
Thanks for the feedback, I've changed the implementation after that.